src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -40,13 +40,24 @@ export default function ColorPickerControl({
 
     const handleClick = (): void => setIsColorPickerToggled(!isColorPickerToggled);
 
+    const handleMouseDown = (event: React.MouseEvent<HTMLButtonElement>): void => {
+        if (event.currentTarget.focus) {
+            // Buttons do not receive focus in Firefox and Safari on MacOS
+            event.currentTarget.focus();
+            // Stop propagation to higher level component which may trigger blur event
```

Comment:
Is this needed? The comment makes it seem uncertain. We should avoid messing with propagation if possible.

Reply:
Yeah, it's needed. The deselect mouse down handler in annotations blurs the button just after it gets focused.

Reply:
Does that mean clicking the color picker button doesn't deselect an active annotation?